src/fdrs_oled.h

```diff
@@ -33,30 +36,102 @@ void draw_OLED_header()
     display.setTextAlignment(TEXT_ALIGN_LEFT);
     display.setFont(ArialMT_Plain_10);
     #endif
+    display.drawHorizontalLine(0, 15, 128);
+    display.drawHorizontalLine(0, 16, 128);
```

Comment:
Note: On dual-color OLEDs, line 15 is yellow and 16 is blue. Logically, line 16 should be drawn in the page since it's the beginning of the large section of the display. /nitpick